DOI.js

```diff
@@ -9,13 +9,14 @@
 	"inRepository": true,
 	"translatorType": 4,
 	"browserSupport": "gcsv",
-	"lastUpdated": "2015-02-12 07:40:24"
+	"lastUpdated": "2016-01-09 12:40:47"
 }
 
 var items = {};
 var selectArray = {};
```

Comment:
Moving "into" the function is not working because we will call `retrieveDois` a second time from inside the `done`-handler. Or do you mean to add it as a parameter to the function? I.e. `retrieveDOIs(DOIs, doc, providers, items, selectArray)`?